runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -318,6 +328,15 @@ public void onItemContainerChanged(final ItemContainerChanged event)
 
 		inventoryItems = event.getItemContainer().getItems();
 
+		for (Item i : inventoryItems)
+		{
+			if (i.getId() == ItemID.RUNE_POUCH || i.getId() == ItemID.RUNE_POUCH_L)
+			{
+				inventoryItems = ArrayUtils.addAll(inventoryItems, getRunepouchContents());
```

Comment:
This doesn't look right to me. If I were to have an inventory containing a law rune, an air rune, and a rune pouch which contains four air runes, I'd get `inventoryItems = [ law, air, air x 4 ]` instead of the expected `[ law, air x 5 ]`

Reply:
Good point, I'll add some logic to combine the items